sql/updates/world/3.3.5/9999_99_99_99_world_burst.sql

```diff
@@ -1,44 +1,26 @@
-(31142,0,7,0,61,0,100,0,     0,0,0,0, 11,59580,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Icy Ghoul - On Spellhit "Burst at the Seams" - Cast "Burst at the Seams"'),
-
-(31147,0,0,1, 8,0,100,0, 59579,0,0,0, 11,60041,2,0,0,0,0, 7,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Assign Geist Kill Credit to Master"'),
-(31147,0,1,2,61,0,100,0,     0,0,0,0, 11,52565,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Troll Explosion"'),
-(31147,0,2,3,61,0,100,0,     0,0,0,0, 11,52520,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Explode Abomination Meat"'),
-(31147,0,3,4,61,0,100,0,     0,0,0,0, 11,52578,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Explode Troll Meat"'),
-(31147,0,4,5,61,0,100,0,     0,0,0,0, 11,52578,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Explode Troll Meat"'),
-(31147,0,5,6,61,0,100,0,     0,0,0,0, 11,52516,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Burst at the Seams: Bone"'),
-(31147,0,6,7,61,0,100,0,     0,0,0,0, 11,52580,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Explode Bloody Troll Meat"'),
-(31147,0,7,0,61,0,100,0,     0,0,0,0, 11,59580,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Vicious Geist - On Spellhit "Burst at the Seams" - Cast "Burst at the Seams"'),
-
-(31205,0,0,1, 8,0,100,0, 59579,0,0,0, 11,60039,2,0,0,0,0, 7,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Assign Skeleton Kill Credit to Master"'),
-(31205,0,1,2,61,0,100,0,     0,0,0,0, 11,52565,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Troll Explosion"'),
-(31205,0,2,3,61,0,100,0,     0,0,0,0, 11,52520,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Explode Abomination Meat"'),
-(31205,0,3,4,61,0,100,0,     0,0,0,0, 11,52578,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Explode Troll Meat"'),
-(31205,0,4,5,61,0,100,0,     0,0,0,0, 11,52578,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Explode Troll Meat"'),
-(31205,0,5,6,61,0,100,0,     0,0,0,0, 11,52516,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Burst at the Seams: Bone"'),
-(31205,0,6,7,61,0,100,0,     0,0,0,0, 11,52580,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Explode Bloody Troll Meat"'),
-(31205,0,7,0,61,0,100,0,     0,0,0,0, 11,59580,2,0,0,0,0, 1,0,0,0,0,0,0,0, 'Risen Alliance Soldier - On Spellhit "Burst at the Seams" - Cast "Burst at the Seams"'),
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (52593, 59579);
```

Comment:
Recommended:
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_q13264_q13276_q13288_q13289_bloated_abom_feign_death', 'spell_q13264_q13276_q13288_q13289_burst_at_the_seams_59579');
```

Reply:
I dunno, it seems kinda weird.
spell_id is the primary key and some spells share script names, so deleting on it would be very unsafe.
Try 
```sql
SELECT * FROM spell_script_names WHERE ScriptName = 'spell_gen_creature_permanent_feign_death';
```